crates/compiler/test_gen/src/gen_num.rs

```diff
@@ -1076,6 +1076,90 @@ fn gen_sub_f64() {
     );
 }
 
+#[test]
+#[cfg(any(feature = "gen-llvm", feature = "gen-dev", feature = "gen-wasm"))]
+fn gen_sub_i8() {
+    assert_evals_to!(
+        indoc!(
+            r#"
+                    1i8 - 2i8 - 3i8
+                "#
```

Comment:
if you write this as just `" 1i8 - 2i8 - 3i8"` then the whole assert should fit on one line, much cleaner.